web/context/FeatureToggle.tsx

```diff
@@ -18,25 +26,49 @@ export default function FeatureToggleWrapper({
 }: {
   children: ReactNode
 }) {
-  const EXPERIMENTAL_FEATURE_ENABLED = 'expermientalFeatureEnabled'
-  const [experimentalEnabed, setExperimentalEnabled] = useState<boolean>(false)
+  const EXPERIMENTAL_FEATURE = 'experimentalFeature'
+  const IGNORE_SSL = 'ignoreSSLFeature'
+  const HTTPS_PROXY_FEATURE = 'httpsProxyFeature'
+  const [experimentalFeature, directSetExperimentalFeature] = useState<boolean>(false)
+  const [ignoreSSL, directSetIgnoreSSL] = useState<boolean>(false)
+  const [proxy, directSetProxy] = useState<string>('')
 
   useEffect(() => {
-    setExperimentalEnabled(
-      localStorage.getItem(EXPERIMENTAL_FEATURE_ENABLED) === 'true'
+    directSetExperimentalFeature(
+      localStorage.getItem(EXPERIMENTAL_FEATURE) === 'true'
+    )
+    directSetIgnoreSSL(
+      localStorage.getItem(IGNORE_SSL) === 'true'
+    )
+    directSetProxy(
+      localStorage.getItem(HTTPS_PROXY_FEATURE)
```

Comment:
@markmehere, that's a really great PR. I'd like to apply this change to fix the compilation error, and then we'll be good to go.
```suggestion
      localStorage.getItem(HTTPS_PROXY_FEATURE) ?? ""
```